libjolie/src/main/java/jolie/lang/parse/OLParser.java

```diff
@@ -3303,13 +3314,19 @@ private OLSyntaxNode parseFactor()
 
 		switch( token.type() ) {
 		case ID:
+			String id = token.content();
 			path = parseVariablePath();
```

Comment:
Mmmh, what happens if one writes `x.y@Hello( "x" )`, which should be forbidden? It looks like you'll parse it incorrectly as x@Hello("x").